### PR TITLE
New share sheet

### PR DESCRIPTION
**Pull request by SaraVieira**
New share sheet with the new components

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/4626/commits/59d11066a2d7c6c197b3f99d4ebb3621f4dfc11e is at https://pr4626.build.csb.dev/s/new.

File: packages/app/src/app/pages/common/Modals/ShareModal/SocialShare.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  getButtonHTML,
+  getButtonMarkdown,
+  getEmbedUrl,
+  getDevToLink,
+  getTwitterLink,
+} from './getCode';
+import { Field } from '.';
+
+export const SocialShare = ({ sandbox, mainModule, state, select }) => (
+  <Collapsible title="Other Share Options">
```
Comment (siddharthkp) [inline packages/app/src/app/pages/common/Modals/ShareModal/SocialShare.tsx:26]:
```suggestion
  <Collapsible title="Other Sharing Options">
```

**Comment by siddharthkp** [comment]
Looking really good ✨ 

Noticed: 
- `Edit on CodeSandbox` has a typo
- The share on twitter, share on dev.to buttons aren't working for me.

**Comment by DannyRuchtie** [comment]
@SaraVieira sorry for the typo in the button totally missed that one

I also noticed a strange line underneath the tab bar in the embed itself. I think this was not there before. Could it be it was introduced with the increase of its height? 

Screenshot 2020-08-04 at 10 07 05

**Comment by SaraVieira** [comment]
@siddharthkp All is fixed! Thank you for the feedback

**Pull request merged**